src/utils/rest-api.ts

```diff
@@ -202,7 +214,42 @@ export function fetchVersion(): Promise<Record<string, any>> {
         });
 }
 
-export function fetchAppsAndUrls(): Promise<Array<Record<string, any>>> {
+export type MetadataCommon = {
+    name: string;
+    url: Url;
+    appColor: string;
+    hiddenInAppsMenu: boolean;
+};
+
+export type MetadataStudy = MetadataCommon & {
+    readonly name: 'Study';
+    resources?: Array<{
+        types: string[];
+        path: string;
+    }>;
+    predefinedEquipmentProperties?: {
+        substation?: {
+            region?: string[];
+            tso?: string[];
+            totallyFree?: any[];
+            Demo?: string[];
+        };
+        load?: {
+            codeOI?: any[];
+        };
+    };
+    defaultParametersValues?: {
+        fluxConvention?: string;
+        enableDeveloperMode?: string; //maybe 'true'|'false' type?
+        mapManualRefresh?: string; //maybe 'true'|'false' type?
```

Comment:
boolean ? If it's parsed as json it will return boolean

Reply:
well in the json they're literal string `"true"` and `"false"`, but dunno what is the strict rule with these field